script.video.bgtime_tv/addon.xml

```diff
@@ -29,6 +28,6 @@
 		</disclaimer>
 		<website>https://bgtime.tv/</website>
 		<email>support@mytv.bg</email>
-		<news>v1.0.3 created by mariapmarkova@gmail.com</news>
+		<news>v1.0.7 created by mariapmarkova@gmail.com</news>
```

Comment:
this is not the use for the news element :)
here you can simply share what's new in the latest version